app/src/organisms/ProtocolDetails/ProtocolLiquidsDetails.tsx

```diff
@@ -34,22 +45,45 @@ export const ProtocolLiquidsDetails = (
       overflowY="auto"
       data-testid="LiquidsDetailsTab"
     >
-      {liquidsInLoadOrder?.map((liquid, index) => {
-        return (
-          <React.Fragment key={liquid.id}>
-            <Flex flexDirection={DIRECTION_COLUMN} marginY={SPACING.spacing16}>
-              <LiquidsListItemDetails
-                liquidId={liquid.id}
-                displayColor={liquid.displayColor}
-                displayName={liquid.displayName}
-                description={liquid.description}
-                labwareByLiquidId={labwareByLiquidId}
-              />
-            </Flex>
-            {index < liquidsInLoadOrder.length - 1 && <Divider />}
-          </React.Fragment>
-        )
-      })}
+      {liquids.length > 0 ? (
+        liquidsInLoadOrder?.map((liquid, index) => {
+          return (
+            <React.Fragment key={liquid.id}>
+              <Flex
+                flexDirection={DIRECTION_COLUMN}
+                marginY={SPACING.spacing16}
+              >
+                <LiquidsListItemDetails
+                  liquidId={liquid.id}
+                  displayColor={liquid.displayColor}
+                  displayName={liquid.displayName}
+                  description={liquid.description}
+                  labwareByLiquidId={labwareByLiquidId}
+                />
+              </Flex>
+              {index < liquidsInLoadOrder.length - 1 && <Divider />}
+            </React.Fragment>
+          )
+        })
+      ) : (
+        <Flex
+          padding={` ${SPACING.spacing16} ${SPACING.spacing16} ${SPACING.spacing32} ${SPACING.spacing16}`}
```

Comment:
ProtocolDetails has the following padding so for this part, we don't need padding for left and right.
```ts
              padding={`${SPACING.spacing16} ${SPACING.spacing16} 0 ${SPACING.spacing16}`}

```

```suggestion
          paddingTop={SPACING.spacing16}
          paddingBottom={SPACING.spacing32}
```

Reply:
https://www.figma.com/file/0hYQ4lFJbAAI33jQMRt1Di?node-id=1836:44526&mode=dev#502945950